packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -242,10 +245,10 @@ const renderWithRedux = (
             )}`
           );
 
-        const { response, error } = clonedSdkMocks[index];
+        const { response, error } = sdkMocksStack[index];
 
         // remove result as each result is only mocked once
-        sdkMocks.splice(index, 1);
+        sdkMocksStack.splice(index, 1);
```

Comment:
I was mutating the user-passed thing instead of the clone. This resulted in the error not showing the `sdkMocks` provided by the user but by showing the mocks at that point in time instead which made debugging a bit harder.